### PR TITLE
Add documentMatches to Storefront KYC resource

### PR DESCRIPTION
**Pull request by colinbird**
Add `documentMatches` to response for StorefrontGetKycDocument

https://preview.redoc.ly/rebilly-storefront/kyc-storefront-document-matches/tag/KYC-Documents#operation/StorefrontGetKycDocument

### CODE REVIEW

**Review by mattsanf — DISMISSED** [review]
Looks good from here https://preview.redoc.ly/rebilly-storefront/kyc-storefront-document-matches/tag/KYC-Documents#operation/StorefrontGetKycDocument

**Comment by ashkarpetin** [comment]
Please put `_links` at the end

File: openapi/components/schemas/Storefront/KycDocuments/ProofOfPurchase.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+allOf:
+  - $ref: ../../Common/CommonKycDocument.yaml
+  - properties:
+      documentMatches:
+        type: object
+        readOnly: true
+        properties:
+          data:
+            $ref: ../../KycDocument/PurchaseMatches.yaml
+
```
Comment (ashkarpetin) [inline openapi/components/schemas/Storefront/KycDocuments/ProofOfPurchase.yaml:10]:
```suggestion
```

File: openapi/components/schemas/Common/CommonKycDocument.yaml [hunk 2]
```diff
@@ -65,6 +65,9 @@ properties:
     description: Processing date/time.
     allOf:
       - $ref: ../ServerTimestamp.yaml
+  documentMatches:
+    type: object
+    readOnly: true
```
Comment (ashkarpetin) [inline openapi/components/schemas/Common/CommonKycDocument.yaml:70]: I wonder what is empty object means?

Comment (colinbird) [reply openapi/components/schemas/Common/CommonKycDocument.yaml:70]: I put it there to try to order the properties better, it can be removed

Comment (colinbird) [reply openapi/components/schemas/Common/CommonKycDocument.yaml]: I removed it

**Review by ashkarpetin — APPROVED** [review]

**Review by ninjasimon — APPROVED** [review]

**Review by mattsanf — APPROVED** [review]

**Pull request merged**